app/assets/stylesheets/variables.scss

```diff
@@ -13,7 +13,7 @@ $headings-font-weight: 300;
 $font-size-base: 14px;
 $line-height-base: 1.428571429;
 $font-family-base: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
-
+$screen-sm-min: 480px;
```

Comment:
Would it make sense to add a comment here about why we're changing the default, or is the commit message sufficient?